<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="inputFileButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Import form file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printTableButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Print Table</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tableOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dataLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>391</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addElementButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeElementButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportToFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Export to file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="indexLineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="printTreeButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Print Tree</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="treeOutput">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>9</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="findElementButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exampleInputButton">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>460</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Example input</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="elementListOutput">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="debugOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>391</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setSizeButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Clear Table</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
